stage/traefik/conf/dynamic.yml

```diff
@@ -0,0 +1,85 @@
+http:
+  routers:
+    # staging.wetfish.net - Wetfish Home Page
+    staging-wetfish-home:
+      entrypoints:
+        - "websecure"
+      rule: "Host(`staging.wetfish.net`)"
+      tls:
+        certResolver: letsencrypt
+        options: mintlsversion
+      middlewares:
+        - "ipwhitelist-cloudflare"
+      service: "staging-wetfish-home"
```

Comment:
The service name should actually be `wetfish-home` since we will no longer be using separate dockerfiles for staging.